semver.py

```diff
@@ -658,30 +729,48 @@ def bump_build(version, token="build"):
     >>> semver.bump_build('3.4.5-rc.1+build.9')
     '3.4.5-rc.1+build.10'
     """
-    verinfo = parse(version)
-    verinfo["build"] = _increment_string(verinfo["build"] or (token or "build") + ".0")
-    return format_version(
-        verinfo["major"],
-        verinfo["minor"],
-        verinfo["patch"],
-        verinfo["prerelease"],
-        verinfo["build"],
-    )
+    return str(VersionInfo.parse(version).bump_build(token))
 
 
+@deprecated()
 def finalize_version(version):
     """
     Remove any prerelease and build metadata from the version.
 
+    .. deprecated:: 2.9.2
+       Use :func:`semver.VersionInfo.bump_format_version` instead.
```

Comment:
There's no `bump_format_version`

Reply:
Right, it was a bad typo. I'm wondering what we should do with `semver.finalize_version`. Probably, there should be a `semver.VersionInfo.finalize_version` method, right?
Or do you see another option?

Reply:
After thinking about this, I guess a `semver.VersionInfo.finalize_version` would be helpful. I think, the signature would be:

```python
def finalize_version(self):
    ...
```

However, I'm not sure about what to return:

1. Set prerelease and build to None and return `self`.
2. Create a new instance with major, minor, and patch. So efficiently, it does not touch the current version.

What do you think should be the preferred way? (IMHO, I tend to option 1)

Reply:
After a second thought, I've implemented option 2. That would be consistent with, for example, the `bump_*` methods. I've marked this discussion as resolved.